criu/proc_parse.c

```diff
@@ -593,14 +593,49 @@ static int handle_vma(pid_t pid, struct vma_area *vma_area,
 		}
 	} else if (*vm_file_fd >= 0) {
 		struct stat *st_buf = vma_area->vmst;
+		struct stat st_kfd, st_dri_min;
+		char img_path[128];
+		int ret;
 
 		if (S_ISREG(st_buf->st_mode))
 			/* regular file mapping -- supported */;
 		else if (S_ISCHR(st_buf->st_mode) && (st_buf->st_rdev == DEVZERO))
 			/* devzero mapping -- also makes sense */;
 		else {
-			pr_err("Can't handle non-regular mapping on %d's map %"PRIx64"\n", pid, vma_area->e->start);
-			goto err;
+			ret = stat("/dev/kfd", &st_kfd);
+			if (ret == -1) {
+				pr_perror("fstat error for /dev/kfd\n");
```

Comment:
In pr_perror() calls everywhere please remove excess newline specifiers '\n', pr_perror() adds '\n' by itself.

Reply:
Ack